msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -444,12 +455,14 @@ public static SignUpSubmitCodeCommandParameters createSignUpSubmitCodeCommandPar
      * @param configuration PCA configuration
      * @param tokenCache token cache for storing results
      * @param continuationToken Continuation token received from the start command
+     * @param correlationId correlation ID to use in the API request, taken from the previous API response in the flow
      * @return Command parameter object
      * @throws ClientException
      */
     public static SignUpResendCodeCommandParameters createSignUpResendCodeCommandParameters(
             @NonNull final NativeAuthPublicClientApplicationConfiguration configuration,
             @NonNull final OAuth2TokenCache tokenCache,
+            @NonNull final String correlationId,
```

Comment:
`correlationid` should come after `continuatioToken` as in other methods in this file